apps/condo/domains/notification/schema/MessageBlackList.test.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Generated by `createschema notification.MessageBlackList 'organization?:Relationship:Organization:SET_NULL; user?:Relationship:User:CASCADE; phone?:Text; email?:Text;'`
+ */
+
+const faker = require('faker')
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE, waitFor } = require('@core/keystone/test.utils')
+
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj, expectToThrowAccessDeniedErrorToObjects,
+} = require('@condo/domains/common/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser, createTestEmail, createTestPhone } = require('@condo/domains/user/utils/testSchema')
+const { MessageBlackList, createTestMessageBlackList, updateTestMessageBlackList, Message } = require('@condo/domains/notification/utils/testSchema')
+const { createTestBillingIntegrationLog } = require('@condo/domains/billing/utils/testSchema')
+const { makeClientWithRegisteredOrganization, inviteNewOrganizationEmployee, reInviteNewOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema/Organization')
+const { DIRTY_INVITE_NEW_EMPLOYEE_MESSAGE_TYPE, MESSAGE_SENT_STATUS, EMAIL_TRANSPORT, MESSAGE_ERROR_STATUS } = require('@condo/domains/notification/constants/constants')
+
+describe('MessageBlackList', () => {
+    describe('accesses', () => {
+        describe('create', () => {
+            it('support can create MessageBlackList', async () => {
```

Comment:
Optional: Make it readable in natural language format, like
"test support can create…"
You have now:
"it support can create"

```suggestion
            test('support can create MessageBlackList', async () => {
```

An so on for others.